server/jetstream_api.go

```diff
@@ -3324,6 +3315,17 @@ func (s *Server) jsConsumerCreate(sub *subscription, c *client, a *Account, subj
 	}
 
 	if s.JetStreamIsClustered() && !req.Config.Direct {
```

Comment:
Had to check again that s.JetStreamIsClustered() cannot change its evaluation (js.cluster seem to never been reset to nil)... almost wonder if we should not "snapshot" that this is clustered the first time we have the `if s.JetStreamIsClustered()` above so we have the guarantee that `js` won't be nil here..

Reply:
changed